gwswlib/tests/test_exporter.py

```diff
@@ -26,6 +28,78 @@ def test_check_connection_db(caplog):
     assert session is not None
 
 
+def test_import_hydx_unknown_connection_types(caplog):
+    # TODO ik zou graag in één keer die hydx.connections willen maken zonder .import_csvline te gebruiken. Is dat mogelijk?
```

Comment:
Als je op de `Connection` class de `import_csvline(self, csvline)` als class method gebruikt (een soort __init__(), zoals ik hierboven ergens zei), dan geeft die methode een `Connection` instantie terug.

Qua naamgeving is het helemaal geen raar idee daar dan een tweede classmethod naast te zetten: `import_csvfile(cls, csvfile_ofzo)` Die geeft dan geen losse Connection instantie terug, maar een lijstje van instances.

Als ik naar deze test kijk ziet er wat raar uit dat je een Hydx object aan moet maken om een Connection in te lezen. Dat is wat jij ook bedoelt, toch?

**Perfect:** als de test wat onhandig is, wijst dat erop dat de code nog niet optimaal is. Een groot voordeel van het schrijven van tests: dit soort dingen komen erdoor bovenborrelen.

In dit geval is de verantwoordelijkheid tussen Hydx en Connection misschien wat onduidelijk?